lib/netdev-tc-offloads.c

```diff
@@ -1283,7 +1283,7 @@ netdev_tc_flow_put(struct netdev *netdev, struct match *match,
     }
 
     NL_ATTR_FOR_EACH(nla, left, actions, actions_len) {
-        if (flower.action_count >= TCA_ACT_MAX_PRIO) {
+        if (flower.action_count >= 16) {
             VLOG_DBG_RL(&rl, "Can only support %d actions", flower.action_count);
```

Comment:
can u do another commit for this series to fix the log. it should print the define and not what the user chose.

Reply:
sorry its not a bug, just looks like this. because action_count changes here. but it will be readable to use the define in the print.